service/src/main/java/io/camunda/service/search/sort/UserTaskSort.java

```diff
@@ -35,6 +35,11 @@ public Builder completionDate() {
       return this;
     }
 
+    public Builder priority() {
+      currentOrdering = new FieldSorting("priority", null);
```

Comment:
💭  Reminding me to create a follow up issue and add the priority fields for Client/Rest API 

As this is on the review, wont add it now to avoid large scope

Reply:
It's part of https://github.com/camunda/camunda/issues/19833 for the Tasklist implementation, will be handled by me later on :)